app/models.py

```
@@ -279,4 +295,21 @@ class TrainingAndLearning(Document):
+    day= StringField(required=True)  # E.g., "Day 1", "Day 2"
+    date_time= DateTimeField(required=True)  # Date and time of the recording
+    description = StringField()
+    video_url = StringField(required=True)  # URL to the recorded video
+    pdf_name=StringField()  # Duration of the recording in minutes
+    Pdf_url = StringField()  # URL to the associated PDF document
+    created_at = DateTimeField(default=ist_now)
+    updated_at = DateTimeField(default=ist_now)
+    meta = {
+        "indexes": ["employee", "meeting_record_id"],
+        "ordering": ["-created_at"]
     }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)